Shaders/PostProcess/ScreenSpaceReflection/private/ComputeTemporalAccumulation.fx

```diff
@@ -1,9 +1,16 @@
 #include "ScreenSpaceReflectionStructures.fxh"
 #include "SSR_Common.fxh"
+#include "BasicStructures.fxh"
 #include "FullScreenTriangleVSOutput.fxh"
 
 #pragma warning(disable : 3078)
 
+cbuffer cbCameraAttribs
+{
+    CameraAttribs g_CameraCurrAttibs;
```

Comment:
`g_Camera` and `g_PrevCamera` will be more concise and easier to read.

Reply:
Renamed to 'g_CurrCamera` and 'g_PrevCamera`